### PR TITLE
Ray lookAt function

### PR DESCRIPTION
**Pull request by ghost**
I have added a Ray .lookAt function to be able to target a vector in 3D space.

### CODE REVIEW

File: src/math/Ray.js [hunk 1]
```diff
@@ -45,6 +45,12 @@ THREE.Ray.prototype = {
 
 	},
 
+	lookAt: function ( v ) {
+
+		this.direction = v.clone().sub( this.origin ).normalize();
```
Comment (mrdoob) [inline src/math/Ray.js]: How about `this.direction.copy( v ).sub( this.origin ).normalize()`?

Comment (xprogram) [inline src/math/Ray.js]: Yeah, that's better instead of completly reassigning the direction.

**Comment by WestLangley** [comment]
Thanks for your beautification. That should be in a separate PR, actually.

Also, if you don't mind, the new classes/methods you created should be in a separate PR so they can be discussed.

Here, what is important is the new `Ray` method -- and updated Ray doccumentation.

[How to Contribute to three.js](https://github.com/mrdoob/three.js/wiki/How-to-contribute-to-three.js).

**Comment by xprogram** [comment]
Um... I did some commits to my fork and they were automatically added to this PR. I don't know what happened.

**Comment by tomka** [comment]
@xprogram You used the dev branch of your fork for this PR (see just below the title above) and so all commits you add to this branch will show up here as well---PRs refer to a branch, not a commit. That's why it is nicer (and easier to manage) to use a separate branch per feature/PR, just like the second point of the contribution guidelines linked above says. That way you could continue working in other branches (like dev) without having new commits show up in your PRs.

**Comment by mrdoob** [comment]
Oh noes... Do you mind doing a new PR only with the Ray lookat change?

**Comment by xprogram** [comment]
Yeah, sure... closing this now. Third time's the charm, right?

**Pull request closed without merging**